Makefile

```diff
@@ -77,6 +76,10 @@ all: coverage-instrument
 	$(GPRBUILD) -P gnat/lsp_server.gpr -p $(COVERAGE_BUILD_FLAGS) \
 		-XVERSION=$(TRAVIS_TAG)
 	$(GPRBUILD) -P gnat/codec_test.gpr -p $(COVERAGE_BUILD_FLAGS)
+	$(GPRBUILD) -P gnat/lsp_client.gpr -p $(COVERAGE_BUILD_FLAGS) \
+		-XVERSION=$(TRAVIS_TAG)
+	$(GPRBUILD) -P gnat/lsp_client_glib.gpr -p $(COVERAGE_BUILD_FLAGS) \
```

Comment:
We'd rather not build the _glib client libraries systematically - they are really only useful for integration in GNAT Studio and we might move the code there in the future, keeping the Ada Language Server buildable/testable without the dependency on GtkAda.

Reply:
OK, change pushed.
Noted that the Makefile should contain something to ensure the `generate` target gets run.
Noted that the README should really be revised.

Reply:
Thank you for the change, and I do agree the README is due for a refresh.